pkgs/by-name/my/myanon/package.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, autoconf
+, automake
+, flex
+, bison
+}:
+
+stdenv.mkDerivation rec {
+  src = fetchFromGitHub {
+    owner = "ppomes";
+    repo = "myanon";
+    rev = "4d67c0a3da59dc5b8ccf77f82405fd370e0d1cc5";
+    hash = "sha256-A4Qg8RvpkKgXoGTidshUyfX6FnhPR2ih1Nyne8Y87HA=";
+  };
+  pname = "myanon";
+  version = "0.5";
+
+  nativeBuildInputs = [ autoconf automake flex bison ];
+
+  configurePhase = ''
+    ./autogen.sh
+    ./configure --prefix=$out
+  '';
+  buildPhase = ''
+    make
+  '';
+  installPhase = ''
+    make install
+  '';
+
+  meta = with lib; {
```

Comment:
Missing `meta.mainProgram`.

Reply:
Thanks for your review! I'm doing the changes you suggested.
I just don't understand this specific comment, I think I have meta.mainProgram which seems to be defined?